ui_test.go

```diff
@@ -171,6 +161,16 @@ func TestPrefixedUiWarn(t *testing.T) {
 	}
 }
 
+func TestAdvancedUi_Print(t *testing.T) {
+	writer := new(bytes.Buffer)
+	ui := &BasicUi{Writer: writer}
+	ui.WriteString("HELLO")
+
+	if writer.String() != "HELLO" {
+		t.Fatalf("bad: %s", writer.String())
+	}
+}
```

Comment:
Can you add a test so that we capture the output with and without the `StringPrefix` set?